Lib/test/test_pyrepl/test_utils.py

```diff
@@ -60,3 +60,24 @@ def gen_raise():
         self.assertEqual(next(pnw), (3, 4, None))
         with self.assertRaises(ZeroDivisionError):
             next(pnw)
+
+    def test_gen_colors_keyword_highlighting(self):
+        cases = [
+            # no highlights
+            ("a.set", [(".", "op")]),
+            ("obj.list", [(".", "op")]),
+            ("obj.match", [(".", "op")]),
+            ("b. \\\n format", [(".", "op")]),
+            # highlights
+            ("set", [("set", "builtin")]),
+            ("list", [("list", "builtin")]),
```

Comment:
Let's add a highlight test case when we don't have a `.` before as well.

Reply:
Done